addons/website_blog/controllers/main.py

```diff
@@ -152,13 +152,13 @@ def _prepare_blog_values(self, blogs, blog=False, date_begin=False, date_end=Fal
         return {
             'date_begin': date_begin,
             'date_end': date_end,
-            'first_post': first_post.with_prefetch(post_ids),
+            'first_post': tools.lazy(lambda: first_post.with_prefetch(post_ids)),
             'other_tags': other_tags,
             'tag_category': tag_category,
-            'nav_list': self.nav_list(),
+            'nav_list': tools.lazy(self.nav_list),
             'tags_list': self.tags_list,
             'pager': pager,
-            'posts': posts.with_prefetch(post_ids),
+            'posts': tools.lazy(lambda: posts.with_prefetch(post_ids)),
```

Comment:
could we verify/comment why a lazy is needed for the `with_prefetch()` cases? seems like it is still lazy by default